### PR TITLE
Update ZHA reporting issues instructions

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Update ZHA reporting issues instructions, (which did not contain any information on where to report issues with ZHA).

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -611,13 +611,19 @@ LQI (Link Quality Index) values can be hard to interpret for Zigbee. This is bec
 
 ### Reporting issues
 
-When reporting issues, please provide the following information in addition to information requested by issue template:
+If you found what looks to be a bug with the underlying component for the ZHA integration, please report it to the [Issues tracker for Home Assistant Core](https://github.com/home-assistant/core/issues). Or, if you found what looks to be a bug with the UI (User Interface) or Dashboards for ZHA, please report that to the [Issues tracker for Home Assistant Frontend]([https://github.com/home-assistant/core/issues](https://github.com/home-assistant/frontend/issues)).
```
Comment (frenck) [inline source/_integrations/zha.markdown]: This is not information integrations needs to document.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
I'm not in favor of adding this at all. We have centralized instructions on where to go for issues. Having each integration adding their own versions, makes it really hard to maintain and hard to be consistent.

**Comment by Hedda** [comment]
> I'm not in favor of adding this at all. We have centralized instructions on where to go for issues.

OK, I have now removed all information that are in centralized instructions with a link to it instead.

**Review by frenck — APPROVED** [review]
Thanks, @Hedda 👍 

../Frenck

**Pull request merged**